docs/05-technical-reference/00-custom-resources/telemetry-01-logpipeline.md

```diff
@@ -61,67 +61,85 @@ For further LogPipeline examples, see the [samples](https://github.com/kyma-proj
+| **spec.output.grafana-loki.url.valueFrom.secretKeyRef.key** |  |
+| **spec.output.grafana-loki.url.valueFrom.secretKeyRef.name** |  |
+| **spec.output.grafana-loki.url.valueFrom.secretKeyRef.namespace** |  |
+| **spec.output.http** | Configures an HTTP-based output compatible with the Fluent Bit HTTP output plugin. |
+| **spec.output.http.compress** | Defines the compression algorithm to use. |
+| **spec.output.http.dedot** | Enables de-dotting of Kubernetes labels and annotations for compatibility with ElasticSearch based backends. Dots (.) will be replaced by underscores (_). |
+| **spec.output.http.format** | Defines the log encoding to be used. Default is json. |
+| **spec.output.http.host** | Defines the host of the HTTP receiver. |
+| **spec.output.http.host.value** |  |
+| **spec.output.http.host.valueFrom** |  |
+| **spec.output.http.host.valueFrom.secretKeyRef** |  |
+| **spec.output.http.host.valueFrom.secretKeyRef.key** |  |
+| **spec.output.http.host.valueFrom.secretKeyRef.name** |  |
+| **spec.output.http.host.valueFrom.secretKeyRef.namespace** |  |
+| **spec.output.http.password** | Defines the basic auth password. |
+| **spec.output.http.password.value** |  |
+| **spec.output.http.password.valueFrom** |  |
+| **spec.output.http.password.valueFrom.secretKeyRef** |  |
+| **spec.output.http.password.valueFrom.secretKeyRef.key** |  |
+| **spec.output.http.password.valueFrom.secretKeyRef.name** |  |
+| **spec.output.http.password.valueFrom.secretKeyRef.namespace** |  |
+| **spec.output.http.port** | Defines the port of the HTTP receiver. Default is 443. |
+| **spec.output.http.tls** | Defines TLS settings for the HTTP connection. |
+| **spec.output.http.tls.disabled** | Disable TLS. |
+| **spec.output.http.tls.skipCertificateValidation** | Disable TLS certificate validation. |
+| **spec.output.http.uri** | Defines the URI of the HTTP receiver. Default is "/". |
+| **spec.output.http.user** | Defines the basic auth user. |
+| **spec.output.http.user.value** |  |
+| **spec.output.http.user.valueFrom** |  |
+| **spec.output.http.user.valueFrom.secretKeyRef** |  |
+| **spec.output.http.user.valueFrom.secretKeyRef.key** |  |
+| **spec.output.http.user.valueFrom.secretKeyRef.name** |  |
+| **spec.output.http.user.valueFrom.secretKeyRef.namespace** |  |
+| **spec.variables** | References a Kubernetes secret that should be provided as environment variable to Fluent Bit |
+| **spec.variables.name** |  |
+| **spec.variables.valueFrom** |  |
+| **spec.variables.valueFrom.secretKeyRef** |  |
+| **spec.variables.valueFrom.secretKeyRef.key** |  |
+| **spec.variables.valueFrom.secretKeyRef.name** |  |
+| **spec.variables.valueFrom.secretKeyRef.namespace** |  |
+| **status.conditions** | LogPipelineCondition contains details for the current condition of this LogPipeline |
+| **status.conditions.lastTransitionTime** |  |
+| **status.conditions.reason** |  |
+| **status.conditions.type** |  |
+| **status.unsupportedMode** |  |<!-- TABLE-END -->
```

Comment:
There are several attributes having no description but they had descriptions before in the MD. Can you please verify the rows and assure that the best of both sources gets used as the new description

Reply:
in a separate PR, include description into CRD from table that used to be here